lib/config/error-codes/index.js

```diff
@@ -130,18 +132,32 @@ function checkDomains(errorCodesFiles) {
       `Error configuration file : Field 'code' must be an integer. (domain: '${domainName}').`);
     assert(
       domain.code >= 0x00 && domain.code <= 0xFF,
-      `Invalid error configuration file : Field 'code' must be between 0 and 255. (domain: '${domainName}').`);
+      `Error configuration file : Field 'code' must be between 0 and 255. (domain: '${domainName}').`);
 
     checkSubdomains(domain);
   }
 }
 
+function loadPluginsError(pluginManifest, pluginsCodes) {
+ 
+  plugins.subdomains[pluginManifest.name] = {
+    code: 
+      pluginsCodes[pluginManifest.name]
+      || 0x00,
```

Comment:
Raw suggestion (open to discussion): Couldn't we use a predictable value with a reasonable probably of uniqueness?
(I am thinking to something like [pearson hashing](https://en.wikipedia.org/wiki/Pearson_hashing))

Reply:
We chose `0x00` for undocumented subdomain, there is no point to make it unique

Reply:
Hum... Wasn't the whole purpose of having some error codes to be able to programmatically identify them?

Reply:
Yes but if its auto-generated its unpredictable, the only case we have `0x00` is when it's undefined.

Reply:
The purpose of hashes *is* to get a predictable output.

Reply:
What i meant is its generated, the code would change (by restarting kuzzle for example).
Plus, all plugins codes are to be unique (except `0x00`) in a given range.
We already thought about generate it if users dont define it :
https://kaliop.slack.com/archives/G4LN18NLQ/p1566564559390200

Reply:
Nope. A hash function is predictable if it uses the same seed. Even if kuzzle restarts, the generated plugin codes would still be the same.

But Ok, sorry, I had missed the discussion and it is true the users can use the `errorName`.
Considering the risk of collision, it is probably not worth it. Let's forget it.